### PR TITLE
ci: move Fedora dependency install to separate script

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related: #81

### CODE REVIEW

File: dependencies_fedora.sh [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	systemd-devel.x86_64 \
+	glibc-devel.x86_64 \
+	dbus-devel.i686 \
+	systemd-devel.i686 \
+	glibc-devel.i686 \
+	openssl-devel \
+	clang \
+	llvm \
+	llvm-devel \
+	keyutils \
```
Comment (jbaublitz) [inline dependencies_fedora.sh:33]: One note is that we should put up an issue to remove this and update the tests to use the Stratis key API once stratis-storage/stratisd#1983 and stratis-storage/stratis-cli#548 are merged.

Comment (mulkieran) [reply dependencies_fedora.sh:33]: > One note is that we should put up an issue to remove this and update the tests to use the Stratis key API once [stratis-storage/stratisd#1983](https://github.com/stratis-storage/stratisd/pull/1983) and [stratis-storage/stratis-cli#548](https://github.com/stratis-storage/stratis-cli/pull/548) are merged.

@jbaublitz Are you able to do that in the PRS that you cite?

Comment (jbaublitz) [reply dependencies_fedora.sh:33]: I should be able to.

Comment (mulkieran) [reply dependencies_fedora.sh:33]: > I should be able to.

Ok! That will be the best way.

**Pull request merged**